src/main/java/study/step3/model/car/Car.java

```diff
@@ -0,0 +1,30 @@
+package study.step3.model.car;
+
+public class Car {
+
+    private int distance;
+
+    private final int CAR_MOVE_CONDITION = 4;
```

Comment:
```suggestion
    private static final int CAR_MOVE_CONDITION = 4;
    
    private int distance;
```
- 상수의 경우 `static`을 지정해주면 좋을 것 같아요.
- 상수를 상단에 먼저 선언해주세요!